.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# See GitHub's docs for more details:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# TypeScript Team
+* @workos-inc/typescript
```

Comment:
This is is public repo but I don't think our teams are public: https://github.com/orgs/workos-inc/teams/typescript

Reply:
Do the teams need to be public for `CODEOWNERS` to work?

Reply:
I don't believe it does. Seems to register correctly:
> bukinoshita approved these changes on behalf of workos-inc/typescript 6 minutes ago

Reply:
Yea, I think this is fine.

GitHub is still aware of the private team even if you aren't part of the @workos-inc org (this is how it looks if you're not signed into GitHub):

<img width="578" alt="Screen Shot 2020-11-17 at 3 47 51 PM" src="https://user-images.githubusercontent.com/1486634/99445960-492cd580-28ec-11eb-8343-e48260de9fc6.png">

Reply:
Yep! Thanks for the questions and responses, y'all. Going to merge :shipit: